scripts/verify

```diff
@@ -183,17 +183,34 @@ if [[ "$istio" = "enabled" ]]; then
   # TODO(#327): Handle multiple ISTIO_ENABLED properties in schema.
   istio_enabled_key="$(extract_schema_key.py \
     --schema_file=/data/schema.yaml \
-    --type ISTIO_ENABLED)" | head -n 1
+    --type=ISTIO_ENABLED \
+    | head -n 1)"
 
   # Stitch in ISTIO_ENABLED parameter if applicable.
-  if [[ ! -z $istio_enabled_key ]]; then
+  if [[ -n $istio_enabled_key ]]; then
     parameters=$(echo "$parameters" \
       | jq \
         --arg istio_enabled_key "$istio_enabled_key" \
         '.[$istio_enabled_key] = true')
   fi
 fi
 
+reporting_secret_key="$(extract_schema_key.py \
+  --schema_file=/data/schema.yaml \
+  --type=REPORTING_SECRET \
+  | head -n 1)"
+
+# Stitch in fake REPORTING_SECRET parameter if applicable.
+if [[ -n "$reporting_secret_key" ]]; then
+  if [[ "$(echo "$parameters" | jq ".$reporting_secret_key")" == "null" ]]; then
+    kubectl apply -f "/data/fake-reporting-secret-manifest.yaml" --namespace="$NAMESPACE"
+    parameters=$(echo "$parameters" \
+      | jq \
+        --arg reporting_secret_key "$reporting_secret_key" \
```

Comment:
I don't understand the purpose of `--arg` here. Is it to work around the single quoted text not expanding env var value?

I think the following should work:

```
| jq "$(printf '.%s = "fake-reporting-secret-manifest"' "$reporting_secret_key")"
```

Tried in a shell the following:

```bash
key="mykey"
$ echo '{"a": "b"}' | jq "$(printf '.%s = "value"' "$key")" 
{
  "a": "b",
  "mykey": "value"
}
```